docs/troubleshooting-guide.md

```diff
@@ -154,6 +175,83 @@ Paste the ssh connection string in a command line, PowerShell or terminal applic
 
 ![Connect with SSH](/docs/screenshots/connectssh.PNG)
 
+### Customize your Cromwell on Azure deployment
+Before deploying, you can choose to customize some input parameters to use existing Azure resources. Example:
+
+```
+.\deploy-cromwell-on-azure.exe --SubscriptionId <Your subscription ID> --RegionName <Your region> --MainIdentifierPrefix <Your string> --VmSize "Standard_D2_v2"
+```
+
+Here is the summary of all configuration parameters:
+
+Configuration   parameter | Has default | Validated | Used by update | Comment
+-- | -- | -- | -- | --
+string   SubscriptionId | N | Y | Y | Always required
+string   RegionName = "westus"; | Y | Y | N | Required for new   install
+string   MainIdentifierPrefix = "coa"; | Y | Y | N |  
+string   VmOsVersion = "18.04-LTS"; | Y | N | N |  
+string   VmSize   = "Standard_D3_v2"; | Y | N | N |  
+string   VnetAddressSpace = "10.0.0.0/24"; | Y | N | N |  
+string   VmUsername = "vmadmin"; | Y | N | Y |  
+string   VmPassword | Y | N | Y | Required for update
+string   ResourceGroupName | Y | Y | Y | Required for update.   If provided for new install, it must already exist.
+string   BatchAccountName | Y | N | N |  
+string   StorageAccountName | Y | N | N |  
+string   NetworkSecurityGroupName | Y | N | N |  
+string   CosmosDbAccountName | Y | N | N |  
+string   ApplicationInsightsAccountName | Y | N | N |  
+string   VmName | Y | N | Y | Required for update   if multiple VMs exist in the resource group
+bool     Silent | Y | Y | Y | Integration testing
+bool     DeleteResourceGroupOnFailure | Y | Y | N | Integration testing
+string   CromwellVersion | Y | N | Y |  
+string   TesImageName | Y | N | Y | Integration testing
+string   TriggerServiceImageName | Y | N | Y | Integration testing
+string   CustomCromwellImagePath | N | N | Y | Development
```

Comment:
I'm not sure the Integration Testing and Development comments are going to make sense to users. I think we need to be more explicit with all of the params.